### DISCUSSION TITLE
Convert ends with ffmpeg error

### DISCUSSION THREAD

**Discussion by uwoehler** [yermak/AudioBookConverter#353] [Q&A] [answered]
I'm trying to convert a bunch of mp3 files to a m4b. Conversion starts, but at the end ffmpeg throws an error:
Stream map '0:v' matches no streams.
How can I solve this?

**Comment by yermak** [comment]
quick fix: add any image as artwork.
Proper fix will be released tomorrow.

**Comment by yermak** [comment, marked as answer]
Duplicates #359
